security/sec-iride/pom.xml

```diff
@@ -73,7 +73,9 @@
 
         <!-- Dependencies -->
         <gs.version>2.8-SNAPSHOT</gs.version>
+        <freeemarker.version>2.3.18</freeemarker.version>
         <spring.version>3.1.4.RELEASE</spring.version>
+        <spring-security.version>3.1.4.RELEASE</spring-security.version>

```

Comment:
No, I mean spring.security.version. If you look @ geoserver root pom.xml for 2.8 spring.security.version is 3.1.0.RELEASE